extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/devmode/console/DevConsoleProcessor.java

```diff
@@ -827,7 +827,7 @@ public Object apply(EvalContext ctx) {
 
             switch (ctxName) {
                 case "srcMainPath": {
-                    return srcMainPath.toAbsolutePath().toString();
+                    return srcMainPath.toAbsolutePath().toString().replace("\\", "/");
```

Comment:
How come this is needed? To handle Windows paths?

Reply:
Exactly.
On windows, the post to `q/dev/io.quarkus.quarkus-vertx-http/openInIDE` contains following parameters.

```
className: org.acme.GreetingResource
lang: java
srcMainPath: C:workspacesaviatoride-opensrcmain
line: 14
```

Reply:
Gotcha, thanks